app/views/feeds/_tabs.html.erb

```diff
@@ -1,5 +1,5 @@
 <% content_for :tabs do %>
-  <div class="list-group d-flex col">
+  <div class="list-group d-flex col prx-fixed-btn-container">
```

Comment:
really all this does is add a very large margin to the bottom so you can still read all the tabs if the Add feed button is overlaying it.